gulpfile.js

```diff
@@ -112,75 +110,44 @@ function jsMinWatch() {
   gulp.watch(srcs, ['js-min']);
 }
 
+
 function jsMin() {
   return bundle('umd', 'false', true, true)
-    .pipe(source(artifactName + '-min.js'))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-    .pipe(sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'));
+      .pipe(source(artifactName + '-min.js'))
+      .pipe(buffer())
+      .pipe(sourcemaps.init({loadMaps: true}))
+      .pipe(sourcemaps.write('.'))
+      .pipe(gulp.dest('dist'));
 }
 
-function jsClosure(done) {
-  var moduleDeclaration = 'goog.module(\'' + googModuleName + '\');';
-
-  return bundle('cjs', 'goog.DEBUG', false, false)
-    .pipe(source(artifactName + '-closure.js'))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-    .pipe(replace(/('|")use strict\1;/, moduleDeclaration))
-    .pipe(replace('Object.defineProperty(exports, \'__esModule\', { value: true });', ''))
-    .pipe(replace(' global ', ' undefined ')) 
-    .pipe(sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'));
-}
 
 function jsCommonJS() {
-  return bundle('cjs', "process.env.NODE_ENV !== 'production'", false, true)
-    .pipe(source(artifactName + '-cjs.js'))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-    .pipe(sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'));
+  return bundle('cjs', 'process.env.NODE_ENV !== \'production\'', false, true)
+      .pipe(source(artifactName + '-cjs.js'))
+      .pipe(buffer())
+      .pipe(sourcemaps.init({loadMaps: true}))
+      .pipe(sourcemaps.write('.'))
+      .pipe(gulp.dest('dist'));
 }
 
 function jsDist() {
   // These must be run serially: clean must complete before any of the js
   // targets run. The js and jsMin targets cannot run in parallel as they both
   // change process.env.NODE_ENV. The CommonJS target could run in parallel
   // with the js and jsMin targets, but currently is not.
-  return clean()
-    .then(jsClosureChecks)
-    .then(jsCommonJS)
-    .then(js)
-    .then(jsMin);
-}
-
-function jsClosureChecks() {
-  return gulp.src(srcs)
-    .pipe(replace(' global ', ' undefined ')) 
-    .pipe(closureCompiler({
-      checks_only: 'true',
-      externs: 'node_externs.js',
-      jscomp_error: 'checkTypes',
-      language_in: 'ECMASCRIPT6_STRICT',
-      warning_level: 'VERBOSE'
-    }));
+  return clean().then(jsCommonJS).then(js).then(jsMin);
 }
 
 gulp.task('clean', clean);
 gulp.task('unit', unit);
 gulp.task('unit-dist', unitDist);
 gulp.task('unit-watch', unitWatch);
-gulp.task('lint', lint);
 gulp.task('js', js);
 gulp.task('js-watch', ['js'], jsWatch);
 gulp.task('js-min', jsMin);
 gulp.task('js-min-watch', ['js-min'], jsMinWatch);
 gulp.task('js-dist', jsDist);
-gulp.task('js-closure', jsClosure);
-gulp.task('js-closure-checks', jsClosureChecks);
-gulp.task('build', ['lint', 'unit'], js);
-gulp.task('dist', ['lint', 'unit-dist'], jsDist);
```

Comment:
Need to keep unit-dist as part of the dist task.

You'll need to update the nodejs version for things to run correctly on Travis with your other changes (i.e. https://github.com/sparhami/incremental-dom/commit/0afb3ed37009166088a8a683da2f22ff9e28c0ce)

Reply:
Done